manifests/init.pp

```diff
@@ -52,6 +52,8 @@
 #   $statsd_forward_port
 #       Set the value of the statsd_forward_port varable. Used to forward all
 #       statsd metrics to another host.
+#   $check_freq
+#       Sets how often checks should be run. By default, set to 15s
```

Comment:
While this is valid, we don't recommend changing the `check_freq`. I think we should remove this parameter.